src/qibolab/transpilers/gate_decompositions.py

```diff
@@ -115,7 +115,7 @@ def translate_gate(gate, native_gates: NativeType):
 onequbit_dec.add(gates.T, [gates.RZ(0, np.pi / 4)])
 onequbit_dec.add(gates.TDG, [gates.RZ(0, -np.pi / 4)])
 onequbit_dec.add(gates.I, [gates.I(0)])
-onequbit_dec.add(gates.Align, [gates.Align(0)])
+onequbit_dec.add(gates.Align, lambda gate: [gates.Align(0, delay=gate.delay)])
```

Comment:
I think this will not behave as expected if the `Align` gate contains more than one qubit, which is allowed in its qibo definition. Actually, the same may be true for the identity (`gates.I`) but I guess it didn't matter since identity does nothing.

One potential solution is to skip `Align` in the transpiler, similarly to what we do for the measurement (line 63 in this file).

Reply:
This is true. Thank you. I have made the change for both `gates.Align` and `gates.I`, since (in theory) one should be able to also apply the identity gate to multiple qubits.